ts/api/ProxyApi.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * Proxy client using apisauce
+ */
+
+import apisauce from "apisauce";
+
+import { apiUrlPrefix as proxyBaseURL } from "../config";
+import { IInstallation, IApiProfile, WithOnlyVersionRequired } from "./types";
+
+const create = () => {
+  const api = apisauce.create({
+    baseURL: proxyBaseURL,
+    timeout: 1500
+  });
+
+  const setBearerToken = (token: string) => {
```

Comment:
Can we pass "api" as a parameter instead of relying on the global var ? (why does it return a promise ?)